targets/frontend/src/pages/alerts/[[...params]].js

```diff
@@ -202,3 +224,15 @@ export function AlertPage() {
 // };
 
 export default withCustomUrqlClient(withUserProvider(AlertPage));
+
+function jsxJoin(array, str) {
```

Comment:
not sure I get what is this for ? Where is the comma in the end ?

Reply:
it is Array.join() but work for array containing jsx